opcua.cpp

```diff
@@ -1040,6 +1042,11 @@ void OPCUA::setTraceFile(const std::string &traceFile)
 {
 	if (traceFile == "True" || traceFile == "true" || traceFile == "TRUE")
```

Comment:
FYI.. I am adding static bool stringToBool(const std::string& str) in PR https://github.com/fledge-iot/fledge/pull/1504/ .